tests/testthat/test_rm.R

```diff
@@ -32,13 +34,139 @@ test_that('basics', {
 #   expect_equal(colnames(tr_res), c("x1", "x2"))
 # })
 
+iris_rec <- recipe(~., data = iris)
 
+iris_train <- iris %>%
+  as_tibble() %>%
+  slice(1:75)
+iris_test <- iris %>%
+  as_tibble() %>%
+  # change the position of the variables
+  select(Species, starts_with("Sepal"), starts_with("Petal")) %>%
+  slice(76:150)
```

Comment:
Consider inlining these global objects into the `test_that()` block that you use them in, or adding a `helper-*.R` file to the `tests/testthat/` directory containing a `generate_iris_test_data()` function that returns a list of these objects.

Reply:
I've put them into the `test_that()` blocks :heavy_check_mark: